_data/collection-index.yml

```
@@ -258,3 +258,8 @@
   contact: https://github.com/mickeahlinder/devcontainer-features/issues
   repository: https://github.com/mickeahlinder/devcontainer-features
   ociReference: ghcr.io/mickeahlinder/devcontainer-features
+- name: IU LaTeX Container Templates
+  maintainer: Torben Wetter
+  contact: https://github.com/TorbenWetter/iu-latex-container-templates/issues
+  repository: https://github.com/TorbenWetter/iu-latex-container-templates
+  ociReference: ghcr.io/TorbenWetter/iu-latex-container-templates
```

```suggestion
  ociReference: ghcr.io/torbenwetter/iu-latex-container-templates
```

The ORAS spec requires the ociReference to be in lower case. Can we update here?
The supporting tools does help convert the cases, however, its better to sync as much as possible.

---

Thank you for the feedback!
I have fixed the conflict and updated the `ociReference `.